shared/contribution-const.js

```diff
@@ -0,0 +1,20 @@
+const ContributionConst = {

```

Comment:
Can we place this constant in the same file with others constants. In think we should keep all constant in `constant.js` file